### PR TITLE
LDAP - changes to store more Org fields

### PR DESCRIPTION
**Pull request by fvanderbiest**
Related with #2146

### CODE REVIEW

File: ldap/georchestra.ldif [hunk 1]
```diff
@@ -221,12 +222,75 @@ member: uid=testuser,ou=users,dc=georchestra,dc=org
...
 
+dn: uid=psc,ou=orgs,dc=georchestra,dc=org
+objectClass: top
+objectClass: person
+objectClass: organizationalPerson
+objectClass: inetOrgPerson
+cn: psc
+sn: psc
+uid: psc
+jpegPhoto:: /9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAgGBQgMFBofBgYHCg0dHhwHBwgMFB0j
```
Comment (pmauduit) [inline ldap/georchestra.ldif]: So you can add a `jpegPhoto` attribute onto a `inetOrgPerson` object, but it seems that the original request was on the organizations objects, right ? (either on the `organization` or the `groupOfMembers` object)

Comment (fvanderbiest) [reply ldap/georchestra.ldif]: There's no `jpegPhoto` attribute on `organizations` or the `groupOfMembers` objects.
The trick here is to reference a `inetOrgPerson` instance via the `seeAlso` attribute of the `groupOfMembers` instance

Comment (pmauduit) [reply ldap/georchestra.ldif]: So, one new object for managing the organization, that would have some consequences also on the console code :(

Comment (fvanderbiest) [reply ldap/georchestra.ldif]: True, this is planned.

**Pull request closed without merging**